docs/graphite.md

```diff
@@ -1,4 +1,4 @@
-# Graphite-api
+# Graphite
```

Comment:
please run a `grep -Ri graphite-api` in the root of the MT repo.  you'll find plenty of places that need to be updated still. like the documentation index in the readme,